### ISSUE TITLE
Nemo RabbitVCS 

### ISSUE THREAD

**Issue by jfabini** [linuxmint/nemo-extensions#287]
Unfortunately the nemo-rabbitvcs plugin disappeared from the official Mint repository with Mint 19 Tara. Its well-working integration was one of the  main reasons why I moved from Kubuntu to Mint 18. 
The README on https://github.com/linuxmint/nemo-extensions (still) lists  nemo-rabbitvcs as supported, commit https://github.com/linuxmint/nemo-extensions/commit/def46ac6989ee460524e96e5ba4c9ddd96f11772 contradicts in that nemo-rabbitvcs has been removed intentionally. However, the rabbitvcs ppa seems to not support bionic nor nemo. 
Seems like a coordination issue - is there a way how the rabbitvcs plugin can be maintained as part of the nemo plugins until the rabbitvcs maintainers adopt the nemo-specific modifications and release their package?

**Comment by leigh123linux** [comment]
Fixed https://github.com/linuxmint/nemo-extensions/commit/6336ad26abc76ac7fbc84e3d4b4add52bf6f04df

**Comment by leigh123linux** [comment]
In fedora rabbitvcs package provides the plugins for file-managers

https://koji.fedoraproject.org/koji/buildinfo?buildID=1039513

**Comment by jfabini** [comment]
Thanks for the pointer. Having your comment as background the https://github.com/linuxmint/nemo-extensions/commit/def46ac6989ee460524e96e5ba4c9ddd96f11772 comment "rabbitvcs people are willing to maintain this themselves" reads for me like the _fedora_ people are willing to maintain. Did anyone talk to the Ubuntu/Mint people? Mint configures Nemo as its default file manager and Mint 19 ships no RabbitVCS plugin package (http://packages.linuxmint.com/list.php?release=Tara#main ). The fedora package is of limited use for Mint installations unless one attempts a manual integration (which may or may not be straight forward, the package consists of one source file but fedora bases its build on rabbitvcs 0.17.1 whereas ubuntu 18.04 ships rabbitvcs 0.16.1). However, manual integration is undoubtedly not the way how Mint is supposed to handle this.
Have there been any contacts/notifications to the Mint developers before deleting the rabbitvcs plugin? 

In other words: any hint or background (nemo-extensions maintainers' perspective on the earlier-quoted commit comment) if anyone/who signalled willingness to maintain/ship nemo-rabbitvcs for Mint?

**Comment by leigh123linux** [comment]
Maybe someone will do a rabbitvcs-0.17 ppa 
Rabbitvcs will probably die completely when we move to python3 for nemo-extenstions.

> Seems like a coordination issue - is there a way how the rabbitvcs plugin can be maintained as part of the nemo plugins until the rabbitvcs maintainers adopt the nemo-specific modifications and release their package?

**Comment by eli-schwartz** [comment]
> Seems like a coordination issue - is there a way how the rabbitvcs plugin can be maintained as part of the nemo plugins until the rabbitvcs maintainers adopt the nemo-specific modifications and release their package?

See https://github.com/rabbitvcs/rabbitvcs/pull/227

If the rabbitvcs package is out of date, that sounds like a rabbitvcs issue, not a nemo-extensions issue.

Both nemo-extensions and rabbitvcs were maintaining the rabbitvcs extension, so I filed a pull request with rabbitvcs to merge changes from here, and then delete it from here, since it results in less work for linuxmint to maintain while at the same time ensuring the rabbitvcs people can better control their experience.

Of note: their version got trivially ported to python2/3 unified support

> Having your comment as background the def46ac comment "rabbitvcs people are willing to maintain this themselves" reads for me like the fedora people are willing to maintain.

No, the rabbitvcs people are maintaining the source code, per their stated willingness to do so which *prompted* my removal from this repository.

Fedora merely packages the work of the rabbitvcs developers.

Ubuntu and Linux Mint also package that work... but more slowly. I guess you could discuss with them, their willingness to do a backports release.

> Have there been any contacts/notifications to the Mint developers before deleting the rabbitvcs plugin?

Yes, the original creator and lead developer of Linux Mint is the one who clicked the delete button.
